python3Packages.elkoep-aio-mqtt: init at 0.1.0b4

JamieMagee opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->
- https://pypi.org/project/elkoep-aio-mqtt/
- https://github.com/epdevlab/elkoep-aio-mqtt
- https://www.home-assistant.io/integrations/inels/

NOTE: I can't run `update-component-packages.py` as it is curretly failing with

```
Traceback (most recent call last):
  File "/home/jamie/src/code/nixpkgs/./pkgs/servers/home-assistant/update-component-packages.py", line 333, in <module>
    main()
    ~~~~^^
  File "/home/jamie/src/code/nixpkgs/./pkgs/servers/home-assistant/update-component-packages.py", line 240, in main
    attr_path = name_to_attr_path(name, packages)
  File "/home/jamie/src/code/nixpkgs/./pkgs/servers/home-assistant/update-component-packages.py", line 202, in name_to_attr_path
    assert len(attr_paths) <= 1, f"{req} matches more than one derivation: {attr_paths}"
           ^^^^^^^^^^^^^^^^^^^^
AssertionError: av matches more than one derivation: ['home-assistant.python.pkgs.av', 'home-assistant.python.pkgs.av_13']
```

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

JamieMagee committed:
python3Packages.elkoep-aio-mqtt: init at 0.1.0b4

dotlambda commented on pkgs/development/python-modules/elkoep-aio-mqtt/default.nix:
```diff
@@ -0,0 +1,44 @@
+  meta = {
+    description = "Python library for iNELS mqtt protocol";
+    homepage = "https://github.com/epdevlab/elkoep-aio-mqtt";
+    changelog = "https://github.com/epdevlab/elkoep-aio-mqtt/releases";
```
Why not
```suggestion
    changelog = "https://github.com/epdevlab/elkoep-aio-mqtt/releases/tag/${src.tag}";
```
?

dotlambda committed:
home-assistant: fix update-component-packages.py

We don't want to use the versioned attribute python3Packages.av_13 even
though we override av to be that package.

dotlambda committed:
home-assistant: support inels component

JamieMagee commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 462161`
Commit: `59f81d0a8ff6f47f91d5131a4fe87c518270b1c5`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.inels</li>
    <li>python312Packages.elkoep-aio-mqtt</li>
    <li>python312Packages.elkoep-aio-mqtt.dist</li>
    <li>python313Packages.elkoep-aio-mqtt</li>
    <li>python313Packages.elkoep-aio-mqtt.dist</li>
  </ul>
</details>
